Source/Shaders/GroundPolylineBatchVS.glsl

```diff
@@ -0,0 +1,21 @@
+attribute vec3 currentPosition;
+attribute vec3 previousPosition;
+attribute vec3 nextPosition;
+attribute vec2 expandAndWidth;
```

Comment:
What do you think about spreading this and `a_batchId` across the `w` components of current, previous, and next?